frontend/src/routes.jsx.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react/addons';
+import Router from 'react-router';
+import App from './components/app.jsx.js';
+import SprintBoard from './components/sprintboard/board.jsx.js';
+
+let Route = Router.Route;
+// -- WE MAY NEED THIS AT SOME LATER POINT FOR A 404 page:
+// let NotFoundRoute = Router.NotFoundRoute;
+
+let routes = (
```

Comment:
Question: how is the default route determined? Is it your `<Link to="sprint">Sprint</Link>` code in app.jsx.js?